tests/unit/MigrationTest.js

```diff
@@ -428,31 +428,6 @@ describe('Migrations', () => {
                     });
                 }));
 
-        it('Should remove any instances of participants found in a report', () =>
-            Onyx.multiSet({
```

Comment:
Why are we removing this block?

Reply:
Oh! Right, it's the same reason as provided here: https://github.com/Expensify/App/pull/32326/files#r1445438321